### DISCUSSION TITLE
Disable zeek when importing pcaps with so-import-pcap

### DISCUSSION THREAD

**Discussion by trappapete** [Security-Onion-Solutions/securityonion#2653] [Unsupported Versions] [answered]
Hi, I am new to Security Onion and I am currently researching in the context of my master thesis regarding IDS and IoT malware. Currently I am testing malware traffic samples against suricata. Since some of the pcaps are quite large i would like to know if there is a way to disable the zeek analysis with "so-import-pcap". I would like save some time since the analysis with zeek takes quite long and needs a lot of ressources. I am currently running latest security-onion import node.

I hope somebody could help me here.

Thanks in advance

**Comment by weslambert** [comment, marked as answer]
This is currently not an option, however, it may be something to consider for the future.
